package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Is this file add intended?

Reply:
Absolutely not... I accidentally executed `npm i` inside the root directory and apparently missed this. Will remove.